monitoring/uss_qualifier/scenarios/faa/uft/message_signing_finalize.py

```diff
@@ -21,15 +21,18 @@ def run(self):
         self.begin_test_step("Signal mock USS")
 
         # TODO: Add call to mock USS to finalize message signing report
+        query = self._mock_uss.stop_msg_sign_recording()
+        self.record_query(query)
+
         with self.check(
             "Successful finalization", participants=[self._mock_uss.participant_id]
         ) as check:
-            if False:  # TODO: Insert appropriate check
+            if query.status_code != 200:  # TODO: Insert appropriate check
```

Comment:
This TODO has been fulfilled so it should be removed

Reply:
Will remove. Thanks.